src/Microsoft.Health.Dicom.Core/Features/Workitem/WorkitemDatasetValidator.cs

```diff
@@ -20,5 +22,37 @@ public void Validate(DicomDataset dataset)
         }
 
         protected abstract void OnValidate(DicomDataset dataset);
+
+
+        protected static void ValidateEmptyValue(DicomDataset dataset, DicomTag tag)
+        {
+            EnsureArg.IsNotNull(dataset, nameof(dataset));
+
+            if (dataset.GetValueCount(tag) > 0)
+            {
+                throw new DatasetValidationException(
+                    FailureReasonCodes.ValidationFailure,
+                    string.Format(
+                        CultureInfo.InvariantCulture,
+                        DicomCoreResource.AttributeMustBeEmpty,
+                        tag));
+            }
+        }
+
+        protected static void ValidateNotPresent(DicomDataset dataset, DicomTag tag)
```

Comment:
Do we need this method? Can't we use ValidateRequirement in DicomDatasetExtensions?

Reply:
This is a different requirement, checking for "Tag should not be present". ValidateRequirement checks for "Tag should be present".